README.md

```diff
@@ -78,6 +80,14 @@ The [`gpio-memory-control`](https://snapcraft.io/docs/gpio-memory-control-interf
 sudo snap connect matter-pi-gpio-commander:gpio-memory-control
 ```
 
+#### Thread 
+The Thread management can be enabled with the help of a running OpenThread Border Router (OTBR) agent reachable via DBus. To do this via the [OTBR snap](https://snapcraft.io/openthread-border-router), install the snap and connect the following interface:
+
+```bash
+sudo snap connect matter-pi-gpio-commander:otbr-dbus-wpan0 \
+                  openthread-border-router:dbus-wpan0
+```
+
```

Comment:
I suggest add the following:

Then, enable Thread via snap option:
```bash
snap set matter-pi-gpio-commander args="--thread`
```

Reply:
The argument examples are defined in a different section. I added the Thread example there.